bip-0048.mediawiki

```diff
@@ -99,21 +99,18 @@ Hardened derivation is used at this level.
 
 ===Script===
 
-This level splits the key space into three separate <code>script_type</code>(s). To provide
+This level splits the key space into two separate <code>script_type</code>(s). To provide
 forward compatibility for future script types this specification can be easily extended.
 
-Currently the only script types covered by this BIP are Native Segwit (p2wsh),
-Nested Segwit (p2sh-p2wsh), and Taproot (p2tr).
+Currently the only script types covered by this BIP are Native Segwit (p2wsh) and
+Nested Segwit (p2sh-p2wsh).
 
 The following path represents Nested Segwit (p2sh-p2wsh) mainnet, account 0:
 <code>1'</code>: Nested Segwit (p2sh-p2wsh) <code>m/48'/0'/0'/1'</code></br>
 
 The following path represents Native Segwit (p2wsh) mainnet, account 0:
 <code>2'</code>: Native Segwit (p2wsh) <code>m/48'/0'/0'/2'</code></br>
 
-The following path represents Taproot (p2tr) mainnet, account 0:
-<code>3'</code>: Taproot (p2tr) <code>m/48'/0'/0'/3'</code></br>
-
 The recommended default for wallets is pay to witness script hash <code>m/48'/0'/0'/2'</code>.
 
 To add new script types submit a PR to this specification and include it in the list above:
```

Comment:
Would it make sense to also remove this part on adding new scripts?

Reply:
Ah, good catch! Thanks.